File: test2.js
function oddNotPrime(n) {
    let numArr = [];
    let result = [];
    let notPrimeOdd = [];
    let count = 1;

    for (let i = 0; i < n; i++) {
        let nums = count++;
        numArr.push(nums);
        if (numArr[i] % 2 !== 0) {
            result.push(numArr[i]);
        }
    }

    for (let i = 0; i < result.length; i++) {
        let primeCheck = true;
        for (let j = 2; j < result[i]; j++) {
            if (result[i] % j === 0) {
                primeCheck = false;
                break;
            }
        }
        if (!primeCheck && result[i] > 1) {
            notPrimeOdd.push(result[i]);
        }
    }

    console.log(notPrimeOdd);
}

oddNotPrime(15);

File: test2.py
def odd_not_prime(n):
    
    num_list = []
    result = []

    for i in range(1, n+1):
        if(i%2 != 0):
            num_list.append(i)

    for i in range(len(num_list)):
        for j in range(len(num_list)):
            if(j != i and j > 0):
                if(num_list[i] % num_list[j] == 0):
                    result.append(num_list[i])
    
    print(num_list)
    print(result)

odd_not_prime(15)
